setup.py

```diff
@@ -43,7 +44,7 @@ def _read_long_description():
         "requests>=2.25,<3",
         "wheel",
         "python-gitlab>=1.10,<3",
-        "tomlkit==0.7.0",
+        "tomlkit==0.7.2",
```

Comment:
FYI this was [pined to 0.7.0](https://github.com/relekang/python-semantic-release/pull/337) as it seems it was [causing a bug ](https://github.com/relekang/python-semantic-release/issues/336) where the pyproject.toml was being wiped. 

Should probably have left a comment or even better write a test reproducing the problem... 😞